lib/active_model/serializer/adapter/json.rb

```diff
@@ -7,7 +7,7 @@ class Json < Base
 
         def serializable_hash(options = nil)
           options ||= {}
-          { root => Attributes.new(serializer).serializable_hash(options) }
+          { root => Attributes.new(serializer, instance_options).serializable_hash(options) }
```

Comment:
I'm not sure why it wasn't passed along before. There are some differences in which options you'd want to send from a collection serializer.  In any case, best not to change that here, as it deserves its own discussion/tests/etct.

The difference

- `instance_options` are the options passed to the initializer
- `options`, if present, are an argument to the method

Previously, instance_options were just `@options`, which I found confusing when `options` was a local and `@options` was set at initialization.  To preserve method argument consistency, when I removed the instance vars, I chose `instance_options` as the name.